Source/missiles.cpp

```diff
@@ -3414,41 +3392,31 @@ void __fastcall AddDisarm(int mi, int sx, int sy, int dx, int dy, int midir, int
 
 void __fastcall AddApoca(int mi, int sx, int sy, int dx, int dy, int midir, int mienemy, int id, int dam)
 {
-	int v9;         // esi
-	int v10;        // eax
-	int v11;        // edx
-	int v12;        // ecx
-	signed int v13; // ebx
-	char *v14;      // edi
+	signed int i;
 
-	v9 = mi;
-	v10 = sx - 8;
-	v11 = sx + 8;
-	missile[v9]._miVar1 = 8;
-	missile[v9]._miVar2 = sy - 8;
-	missile[v9]._miVar3 = sy + 8;
-	missile[v9]._miVar4 = v10;
-	missile[v9]._miVar5 = v11;
-	missile[v9]._miVar6 = v10;
+	missile[mi]._miVar1 = 8;
+	missile[mi]._miVar2 = sy - 8;
+	missile[mi]._miVar3 = sy + 8;
+	missile[mi]._miVar4 = sx - 8;
+	missile[mi]._miVar5 = sx + 8;
+	missile[mi]._miVar6 = sx - 8;
 	if (sy - 8 <= 0)
-		missile[v9]._miVar2 = 1;
-	v12 = 111;
+		missile[mi]._miVar2 = 1;
 	if (sy + 8 >= MAXDUNY)
-		missile[v9]._miVar3 = MAXDUNY - 1;
-	if (v10 <= 0)
-		missile[v9]._miVar4 = 1;
-	if (v11 >= MAXDUNX)
-		missile[v9]._miVar5 = MAXDUNX - 1;
-	v13 = 0;
-	v14 = &plr[id]._pLevel;
-	if (*v14 > 0) {
-		do {
-			missile[v9]._midam += random(67, 6) + 1;
-			++v13;
-		} while (v13 < *v14);
+		missile[mi]._miVar3 = MAXDUNY - 1;
+	if (sx - 8 <= 0)
+		missile[mi]._miVar4 = 1;
+	if (sx + 8 >= MAXDUNX)
+		missile[mi]._miVar5 = MAXDUNX - 1;
+	i = 0;
+	if (plr[id]._pLevel > 0) {

```

Comment:
This is probably part of a for-loop togeather with `i = 0` and the do-while loop below.